dockerfiles/Dockerfile.shellcheck

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch-slim
+
+RUN apt-get update && \
+    apt-get -y install make shellcheck && \
+    apt-get clean
+
+WORKDIR /tmp
```

Comment:
Minor: could we use `/go/src/github.com/docker/cli` here too? It just makes it easier to re-use `MOUNTS` in the `docker.Makefile` if the code is mounted at the same place in all conatiners.